plugin/kprom/kprom.go

```diff
@@ -125,14 +125,21 @@ func (m *Metrics) Handler() http.Handler {
 // This method is meant to be called by the hook system and not by the user
 func (m *Metrics) OnNewClient(client *kgo.Client) {
 	var (
-		factory     = promauto.With(m.cfg.reg)
-		namespace   = m.cfg.namespace
-		subsystem   = m.cfg.subsystem
-		constLabels prometheus.Labels
+		factory   = promauto.With(m.cfg.reg)
+		namespace = m.cfg.namespace
+		subsystem = m.cfg.subsystem
 	)
+	var constLabels prometheus.Labels
```

Comment:
No reason to move this down two lines? Or did you want it to be next to dynamicLabels?